modules/tools/docker-registry/values.yaml.tpl

```diff
@@ -36,3 +38,10 @@ configData:
       threshold: 3
   proxy:
     remoteurl: https://registry-1.docker.io
+resources:
+  requests:
+    cpu: 100m
+    memory: 128Mi
+  limits:
+    cpu: 200m
+    memory: 256Mi
```

Comment:
just setting some sane defaults here because the defaults provided by the namespace module definitely won't be enough.  we can tweak this after load testing